Geometry/TestReference/README.md

```diff
@@ -0,0 +1,3 @@
+# Geometry/TestReference
+
+Data-only package, location to store reference files for geometry unit tests
```

Comment:
Do we really need this package in cmssw?

Reply:
@smuzaffar the unit test is in practice failing because nothing is found:

```
Running script: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-19-2300/src/Geometry/MTDGeometryBuilder/test/runTest.sh
 testing Geometry/MTDGeometryBuilder
===== Test "cmsRun mtd_cfg.py" ====
The output diff is 182:
/usr/bin/zdiff: line 72: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-19-2300/src/Geometry/TestReference/data/mtdGeometryRef.log.gz: No such file or directory
===== Test "cmsRun dd4hep_mtd_cfg.py" ====
The output diff is 182:
/usr/bin/zdiff: line 72: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-19-2300/src/Geometry/TestReference/data/mtdGeometryRef.log.gz: No such file or directory
status = 0

---> test GeometryMTDGeometryBuilderTestDriver succeeded
 
^^^^ End Test GeometryMTDGeometryBuilderTestDriver ^^^^
```

Apart that the test should better fail in this case, my interpretation is that the package is unknown if this is never declared. Or do you have an alternative explanation?